src/menu/menu.scss

```diff
@@ -71,7 +75,7 @@
 
       > .iui-icon {
         @include themed {
-          fill: t(iui-icons-color-primary);
+          fill: t(iui-color-foreground-primary);
```

Comment:
Are these changes intended? Or they were done before using icon colors everywhere and now left?

Reply:
I think these are from before the icon color change PR.

Reply:
So shouldn't we change that back to use icon colors? 😀

Reply:
I've updated all the icon fills to use the icon variables but only within `menu.scss`.